backend/routes/errors.js

```diff
@@ -10,6 +10,9 @@ router.get('/', (req, res) => {
   const order = req.query.order || 'desc';
   const filter = { archived: false };
 
+  req.query.environment ? filter.environment = req.query.environment : '';
```

Comment:
```suggestion
filter.environment = req.query.environment || '';
```

Reply:
Nesse caso, essa refatoração não seria a melhor opção, pois se eu passar ' ' para algum filtro, ele não vai conseguir filtrar os resultados... Eu só posso ter o filtro 'environment' se eu tiver um valor dentro dele...

Reply:
acredito que nesse caso essa refatoração não seria a melhor opção, pois se eu passar ' ' pro filtro de 'environment' ele não vai conseguir encontrar um erro correspondente (pelo menos foi o que eu conclui testando)... pelo o que eu entendi quando eu passo um campo em um filtro ele sempre precisa ser preenchido por um valor "válido"...

Reply:
nesse formato, quando o `req.query.environment` não for um valor válido, ele assume o valor `''`

Reply:
sim, mas aí o filter.environment tenta procurar um environment com '' e não encontra... foi o que aconteceu quando eu teste da maneira que você disse... :(